frontend/src/components/license/LicenseNotification.tsx

```diff
@@ -76,16 +91,9 @@ export const LicenseNotification = observer(() => {
                 Upload license
               </Button>
             )}
-            {soonToExpireLicenses.length > 0 && (
-              <Button variant="outline" size="sm" as="a" target="_blank" href="https://redpanda.com/license-request">
-                Renew license
-              </Button>
-            )}
-            {showEnterpriseFeaturesWarning && (
-              <Button variant="outline" size="sm" as="a" target="_blank" href="https://www.redpanda.com/try-redpanda">
-                Request a trial
-              </Button>
-            )}
```

Comment:
Do we want to keep this or is it intentional to request license?

Reply:
Should we use https://redpanda.com/license-request?

Reply:
I believe this was an explicit request on one of the last calls.